src/machine/uart.go

```diff
@@ -1,4 +1,4 @@
-// +build avr esp nrf sam sifive stm32 k210 nxp
+// +build avr esp nrf sam sifive stm32 k210 nxp,!mimxrt1062
```

Comment:
Maybe it would be better to explicitly include devices instead of explicitly excluding devices?

Reply:
Going forward that seems reasonable. It seems to me kinda risky (and a lot of work) trying to go back and making sure you've accounted for all appropriate devices in existing code like this

Reply:
No, it's not risky at all. The smoke tests will fail if you made a mistake. And in this case, there is only one file to include: the chip in the Teensy 3.6.

```suggestion
// +build avr esp nrf sam sifive stm32 k210 nxpmk66f18
```

Reply:
Oh you are referring to this one board, not all boards included by each of the other build tags too... that makes more sense.

Reply:
Also, I should have realized this is only a temporary build constraint until the UART PR #1454 gets merged, which should probably keep the `nxp` tag, as it applies to both Teensy 3.6 and 4.0